client/verta/verta/registry/_check_model_dependencies.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+"""Utility function for checking model dependencies against an environment
+to identify missing packages."""
+
+import warnings
+from typing import Set, Type
+
+from verta._internal_utils import model_dependencies as md
+from verta._internal_utils._pip_requirements_utils import parse_req_spec
+from verta.environment import Python
+from ._verta_model_base import VertaModelBase
+
+
+def _check_model_dependencies(
+        model_cls: Type[VertaModelBase],
+        environment: Python,
+        raise_for_missing: bool = False,
+        ) -> bool:
+    """Scan for missing dependencies in a model's environment.
+
+    This function attempts to scan the provided model class for 3rd-party (not
+    python standard library) dependencies and identify any packages in the
+    provided environment.
```

Comment:
Should this be something like

```suggestion
    python standard library) dependencies and identify any missing packages in the
    provided environment.
```

since it's specifically about checking for missing packages? Unsure though